products/BellHybrid/apps/application-bell-meditation-timer/presenter/IntervalChimePresenter.cpp

```diff
@@ -27,12 +27,13 @@ namespace app::meditation
 
     std::string IntervalChimePresenter::getCurrentInterval() const
     {
-        // Always set interval as none
+        const auto value = settings->getValue(intervalDBRecordName, settings::SettingsScope::AppLocal);
         for (auto const &option : intervalOptions) {
-            if (option.first == std::chrono::minutes{0}) {
+            if (utils::to_string(option.first.count()) == value) {
                 return option.second;
             }
         }
+        // If not found, return "None"
```

Comment:
How about using `std::find_if`? It is more idiomatic:)

Reply:
I would leave it as it is for now as the majority of functions in IntervalChimePresenter class use ranged for loops anyway :)